src/periodic_elements.json

```diff
@@ -0,0 +1,2505 @@
+{
```

Comment:
Where is this file from? For legal purposes and future update-ability it would be nice to put the source in a "comment" like

`"_source": "https://example.net/data.json"`

Reply:
oh wow that had slipped my mind, thank you! ill put the source for the data.